trl/trainer/dpo_config.py

```diff
@@ -282,7 +284,10 @@ class DPOConfig(TrainingArguments):
             "`transformers.TrainingArguments`."
         },
     )
-    loss_type: str = field(
+    loss_weights: Optional[Dict[str, float]] = field(
+        default_factory=lambda: ["your_values"]
+    )
+    loss_type: List[str] | str = field(
```

Comment:
I need to check if this work with the parser and the cli

Reply:
If you could let me know how you might check it, I could do it and report back to you.

Reply:
running this should work:

```
trl dpo --output_dir {tmp_dir} --model_name_or_path trl-internal-testing/tiny-Qwen2ForCausalLM-2.5 --dataset_name trl-internal-testing/zen --dataset_config standard_preference --report_to none
```

Reply:
I tried the above command, which resulted in an issue with the `self.loss_type` being `None`.

Upon changing the command to `trl dpo --output_dir tmp_dir --model_name_or_path trl-internal-testing/tiny-Qwen2ForCausalLM-2.5 --dataset_name trl-internal-testing/zen --dataset_config standard_preference --report_to none --loss-type sigmoid` it started to train. I am not sure why this happens 🤔